### DISCUSSION TITLE
Is there any kind of "administrative install" for IT admins installing to a terminal server?

### DISCUSSION THREAD

**Discussion by bzuillsmith** [velopack/velopack.docs#11] [Q&A] [answered]
I know this installs per-user. Squirrel could produce a special `.msi` file for administrative installs that was basically installing a script to run the installer when a user logged on and from there we would advise IT admins to run a daily script looking for updates. Is there anything like this msi in velopack?

**Comment by caesay** [comment, marked as answer]
The Squirrel MSI was a huge hack to start with. This will be replaced with https://github.com/velopack/velopack/issues/30 which will be a proper / configurable .msi installer capable of installing a real per-machine app into program files. Until that feature is available, there is no alternative at current unless you want to create and ship your own msi using wix or similar.
